doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -510,7 +510,11 @@ Binary libraries
 Bluetooth libraries and services
 --------------------------------
 
-|no_changes_yet_note|
+* :ref:`hids_readme`:
+
+  * Updated HID boot mouse report length to ``3``.
+    The :c:func:`bt_hids_boot_mouse_inp_rep_send` function allows to provide only button state, horizontal movement and vertical movement.
```

Comment:
```suggestion
    The :c:func:`bt_hids_boot_mouse_inp_rep_send` function allows to provide only the button state and horizontal and vertical movement.
```

Reply:
Slightly updated to `buttons state` (as we might report multiple buttons)